test/specs/commonTests/isConformant.js

```diff
@@ -13,6 +13,7 @@ import hasValidTypings from './hasValidTypings'
  * Assert Component conforms to guidelines that are applicable to all components.
  * @param {React.Component|Function} Component A component that should conform.
  * @param {Object} [options={}]
+ * @param {String[]} [options.disabledHandlers=[]] An array of listeners that are disabled.
```

Comment:
Heads up @layershifter we'll need to undo this sometime soon.  Ignoring events means we aren't testing that all events work correctly for the end user.  I don't think it is high priority, but something to keep on our tech debt list.